Assets/BowAndArrowImproved/Scripts/HitCountHittable.cs

```diff
@@ -1,20 +1,21 @@
-using System;
-using System.Collections;
-using System.Collections.Generic;
 using Unity.XR.CoreUtils;
 using UnityEngine;
 using UnityEngine.Events;
 
 public class HitCountHittable : MonoBehaviour
 {
-    public int health => hitCount;
     
     [SerializeField] private int hitCount;
     [SerializeField] private LayerMask hitMask;
-
+    
     public UnityEvent onDeath;
     public UnityEvent onHit;
 
+    public void SetCount(int baseHitCount)
+    {
+        hitCount = baseHitCount;
+    }
+

```

Comment:
we're only setting health from editor right now, no reason to do it from code, remove this function